docs/validator/economics.md

```diff
@@ -46,22 +46,19 @@ This option is still work in progress, so please follow the [NEP on GitHub](http
 
 ## 2. Nodes infrastructure
 
-Once you obtained your NEAR stake, you need an infrastructure to serve the network securely.
-At the beginning of a new `epoch`, the protocol randomly assigns every validator to one of these roles:
-- Chunk Producer, for collecting transactions and applying them to the state
-- Block Producers, for generating the single blocks containing the current chunks 
-- Fishermen (aka 'hidden' validators), for challenging the state to verify the work of other validators
-
-Whether the case, every validator must provide **availability** and **consistency**.
+Once you obtained your NEAR stake, you need to setup the infrastructure to serve NEAR network.
+At a high level, your nodes must provide:
+- Availability
+- Consistency
 
 ### Availability
 
-The protocol requires a `MINIMUM_THRESHOLD` of validating signatures, proportional to the median chunk/block producer in the `epoch` and the number of your *seats*. If your node falls below `90 %`, it will be *kicked out* at the end of the epoch and you will have to issue the staking `proposal` again.
+The protocol requires your nodes to provide a `MINIMUM_THRESHOLD` of signatures, proportional to the median block producer in the `epoch`, and the number of your *seats*. If your node falls below `90 %`, it will be *kicked out* at the end of the epoch, and you will have to issue a staking `proposal` again.
```

Comment:
This is somewhat misleading. It is better to say that the validators are required to produce a certain number of blocks and will be kicked out otherwise.